src/containers/Experiment/SamplesTable.js

```diff
@@ -88,13 +100,26 @@ export default class SamplesTable extends React.Component {
 
     let offset = page * pageSize;
 
-    const data = await getAllDetailedSamples({
+    let data = await getAllDetailedSamples({
       accessionCodes,
       orderBy,
       offset,
       limit: pageSize
     });
 
+    if (experimentAccessionCodes.length === 1) {
+      const experimentAccessionCode = experimentAccessionCodes[0];
+
+      data = data.map(sample => {
+        return { ...sample, experimentAccessionCode };
+      });
+    } else if (experimentAccessionCodes.length > 1) {
+      data = data.map((sample, i) => {
+        const experimentAccessionCode = experimentAccessionCodes[i];
+        return { ...sample, experimentAccessionCode };
+      });
+    }
```

Comment:
What if `experimentAccessionCodes.length === 0`?

Reply:
Ahh yes, I just added a check to hide the "Add/Remove" column if there's no `experimentAccessionCodes`. 👍 Thanks for the reminder!